chipsec/helper/win/win32helper.py

```diff
@@ -387,15 +390,15 @@ def create(self, start_driver):
                  None, 0, u"", None, None )
             if hs:
                 if logger().DEBUG: logger().log( "[helper] service '{}' created (handle = 0x{:08X})".format(SERVICE_NAME,hs) )
-        except win32service.error as (hr, fn, msg):
-            if (winerror.ERROR_SERVICE_EXISTS == hr):
-                if logger().DEBUG: logger().log( "[helper] service '{}' already exists: {} ({:d})".format(SERVICE_NAME, msg, hr) )
+        except win32service.error as err:
+            if (winerror.ERROR_SERVICE_EXISTS == err.args[0]):
+                if logger().DEBUG: logger().log( "[helper] service '{}' already exists: {} ({:d})".format(SERVICE_NAME, err.args[2], err.args[0]) )
                 try:
                     hs = win32service.OpenService( hscm, SERVICE_NAME, (win32service.SERVICE_QUERY_STATUS|win32service.SERVICE_START|win32service.SERVICE_STOP) ) # SERVICE_ALL_ACCESS
-                except win32service.error as (hr, fn, msg):
-                    handle_winerror(fn, msg, hr)
+                except win32service.error as err:
```

Comment:
please pick a different name for exception in this case (try/except above uses err)